src/golang/cmd/migrator/versions/000018_add_serialization_value_to_param_op/types.go

```diff
@@ -0,0 +1,20 @@
+package _000018_add_serialization_value_to_param_op
+
+import (
+	"database/sql/driver"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/utils"
+)
+
+type Spec struct {
+	Type  string
+	Param map[string]string
```

Comment:
so I understand correctly, even though Spec.Param is defined as a struct with two string fields, we can declare it to be read like a map?

Reply:
Yeah, the database we are working on will hold the spec as a struct like the one below
`{"Type":"param","Param":{"serialization_type":"json","val":"\"Aqueduct\""}}`
where there will be a key for "type" and a key for spec based on the type. that will hold the spec and is another map corresponding to the number of variables in the Param Spec on the service side. so in the migrator we just ensure the spec struct has the correct variable names. In this case we only care about the type key (string) and if the key is param (map[string]string).